avocado/utils/cpu.py

```diff
@@ -90,20 +90,42 @@ def cpu_has_flags(flags):
     return True
 
 
+def get_cpu_version():
+    """
+    Get cpu version
+
+    :returns: string cpu version of given machine
+    E:g:- 'i5-5300U' for intel and 'POWER9' for ibm machines
```

Comment:
As x86 has a more complex cpuinfo representation ([CPUID](https://en.wikipedia.org/wiki/CPUID)) than powerpc, it is not easy to make a unified concept of cpu model things, therefore I'm not sure if it is good to call the model name of x86 cpus (e.g. 'i5-5300U') as "cpu version".

If the function is introduced to serve https://github.com/avocado-framework/avocado-vt/pull/2225 , it is better to make it return the x86 microarchitecture name (e.g. 'Skylake', 'EPYC') because we use that more often in VT context (at least for qemu tests). Anyway, I'm not definitely sure what the good implementation is, I'd like to view other people's opinion, thanks.